src/main/java/uk/gov/hmcts/reform/civil/service/referencedata/LocationReferenceDataService.java

```diff
@@ -26,6 +25,12 @@ public class LocationReferenceDataService {
     public static final String COUNTY_COURT_MONEY_CLAIMS_CENTRE = "County Court Money Claims Centre";
     private final LocationReferenceDataApiClient locationReferenceDataApiClient;
     private final AuthTokenGenerator authTokenGenerator;
+    private static final String COURT_TYPE_ID = "10";
```

Comment:
Why not CIVIL_COURT_TYPE_ID ? I keep asking myself what '10' is

Reply:
field is like that only court_type_id. 10 is belong to civil courts